src/components/CreateCard/Cards.js

```diff
@@ -0,0 +1,126 @@
+import React, {useEffect, useState} from "react";
+import {axiosWithAuth} from "../../utils/axios";
+import {Card, Button, Icon, Modal, Input, Select, Avatar, Spin} from "antd";
+import {getCards, deleteCard, getDeckId} from "../../state/actions/CardAction";
+import {connect} from "react-redux";
+import styles from "./AddCard.module.css";
+import HeaderSearchBar from "../ListDeckInfo/HeaderSearchBar";
+import Dashboard from "../../layout/Dashboard/Dashboard";
+import AddCard from "./AddCard";
+import {EditOutlined, DeleteOutlined} from "@ant-design/icons";
+import UpdateCard from "./UpdateCard";
+
+const {Meta} = Card;
+const Search = Input.Search;
+const Option = Select.Option;
+
+function Cards(props) {
```

Comment:
I'd really appreciate it if you desctructured all the props you need. It makes it much easier to know at a glance what to look for in the component.